src/CbcSolverInterface.jl

```diff
@@ -2,8 +2,15 @@ module CbcMathProgSolverInterface
 
 using Cbc.CbcCInterface
 
-importall MathProgBase.SolverInterface
-
+@static if VERSION < v"0.7.0-DEV.2038"
+    importall MathProgBase.SolverInterface
+else
+    # Emulate importall.
+    import MathProgBase.SolverInterface
+    for name in names(SolverInterface)
+        @eval import MathProgBase.SolverInterface: $(name)
```

Comment:
It would be better not to import all but rather to do https://github.com/JuliaOpt/CSDP.jl/blob/658cf4e8c3d9702b779dce73ae70d23f63096186/src/MPBWrapper.jl#L1-L2
and then prefix MPB functions by `MPB.`
This is much cleaner and more consistent with what we do now in MOI

Reply:
I don't disagree but I will leave that to someone who is familiar with the codebase. The `importall` emulation should be good enough for bridging over to 1.0.